test/polymer_gestures_test.dart

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+@TestOn('browser')
+library polymer_interop.polymer_base_test;
+
+import 'dart:html';
+import 'dart:js';
+
+import 'package:test/test.dart';
+import 'package:web_components/web_components.dart';
+import 'package:polymer_interop/src/polymer_gestures.dart';
+
+void showTestRunnerFrame() {
+  // Make testrunner iFrame visible otherwise transitions not get fired ...
+
+  WindowBase w = window.parent;
+
+  JsObject doc = new JsObject.fromBrowserObject(w)['document'];
+
+  JsObject res = doc.callMethod("querySelector", ['iframe']);
+  res['style']['visibility'] = 'visible';
+}
+
+void hideTestRunnerFrame() {
+  WindowBase w = window.parent;
+
+  JsObject doc = new JsObject.fromBrowserObject(w)['document'];
+
+  JsObject res = doc.callMethod("querySelector", ['iframe']);
+  res['style']['visibility'] = '';
+}
+
+main() async {
+  await initWebComponents();
+  group('simulate events', () {
+    var app;
+    JsObject jsApp;
+
+    setUp(() {
+      app = new Element.tag('x-app');
+      jsApp = new JsObject.fromBrowserObject(app);
+      document.body.children.add(app);
+      showTestRunnerFrame();
+    });
+
+    tearDown(() {
+      document.body.children.remove(app);
```

Comment:
K then, I think it's better to call it in the `tearDown` although I've noted the test runner automatically resets.